src/orbit/lib/arrays.js

```diff
@@ -24,4 +24,16 @@ function some(array, predicate) {
   return false;
 }
 
-export { every, some };
+function firstResult(array, valueFn) {
```

Comment:
It's like `find` but instead of returning the item from the array it returns the result of the `valueFn`. I _could_ implement this with find instead, but then I would have to call the same `valueFn` twice on the same value... Still thinking there must be a more idiomatic way to do this. Will keep pondering.